src/components/Main/Cardlist/Cardlist.jsx

```diff
@@ -0,0 +1,37 @@
+import { useEffect, useState } from "react";
+import Card from "../Card/Card";
+import { getFolderInfo } from "../../../utils/api";
+import "./Cardlist.css";
+
+function Cardlist() {
+  const [card, setCard] = useState([]);
+  const [createdAt, setCreatedAt] = useState([]);
+  const [url, setUrl] = useState("");
+
+  useEffect(() => {
+    const fetchFolderDate = async () => {
+      try {
+        const data = await getFolderInfo();
+
+        setCard(data.folder.links);
+        setCreatedAt(data.folder.links.createdAt);
+        setUrl(data.folder.links.url);
+      } catch (error) {
+        console.log(error);
+      }
+    };
+    fetchFolderDate();
+  });
+
+  return (
+    <section className="container">
+      <ul className="cardlist">
+        {card.map((data) => {
+          return <Card key={data.id} data={data} />;
+        })}
```

Comment:
```suggestion
        {card.map((data) => (
          <Card key={data.id} data={data} />
        ))}
```
[중요도: 중간] 굳이 return문이 필요해보이지는 않네요.
key를 data값의 id로 선정해 identify를 유지한 것 좋습니다!

Reply:
감사합니다!
return을 없앴더니 map에서 만들어지는 값이 없어 Card 컴포넌트가 생성되지 않아 어쩔수없이 붙였습니다.. 🥹 혹시 더 좋은 방법이 있을까요?

Reply:
음... 현재로써는 더 좋은 방법이 생각나지는 않는군요 ㅎㅎ